code/modules/mob/living/simple_animal/hostile/netherworld.dm

```diff
@@ -101,11 +126,44 @@
 		if(M)
 			playsound(src, 'sound/magic/demon_consume.ogg', 50, TRUE)
 			M.adjustBruteLoss(60)
-			new /obj/effect/gibspawner/generic(get_turf(M), M)
 			if(M.stat == DEAD)
 				var/mob/living/simple_animal/hostile/netherworld/blankbody/blank
 				blank = new(loc)
-				blank.name = "[M]"
-				blank.desc = "It's [M], but [M.p_their()] flesh has an ashy texture, and [M.p_their()] face is featureless save an eerie smile."
-				visible_message("<span class='warning'>[M] reemerges from the link!</span>")
-				qdel(M)
\ No newline at end of file
+				blank.name = "[M.real_name]"
+				blank.desc = "It's [M.real_name], but [M.p_their()] flesh has an ashy texture, and [M.p_their()] face is featureless save an eerie smile."
+				blank.maxHealth = 150
+				blank.health = 150
+				blank.melee_damage_upper = 15
+				visible_message("<span class='warning'>[M.real_name] reemerges from the link!</span>")
+				if((blankckey) && (blankckey == M.ckey))
+					blank.ckey = blankckey
+					blankckey = null
+					to_chat(blank, "<span class='biggerdanger'>Hatred courses through you... you can't give up your spirit... not yet. Help your newfound compatriots complete their objectives!</span>")
+					SSticker.mode.add_cultist(blank.mind, 1)
+					blank.mind.special_role = "Cultist"
+				qdel(M)
+				blankckey = null
+
+/*/obj/structure/spawner/nether/attackby(obj/item/W as obj, mob/user as mob, params)
+	if(istype(W, /obj/item/grab) && get_dist(src,user)<2)
+		if(iscultist(user))
+			var/obj/item/grab/G = W
+			if(iscultist(G.affecting))
+				return
+			if(!ishuman(G.affecting))
+				return
+			if(!G.affecting.ckey)
+				return
+			if(G.state<3)
+				to_chat(user, "<span class='warning'>You need a better grip to do that!</span>")
+				return
+			else
+				G.affecting.loc = src.loc
+				G.affecting.Weaken(5)
+				visible_message("<span class='warning'>[G.assailant] shoves [G.affecting] into the portal!</span>")
+				contents.Add(G.affecting)
+				blankckey = G.affecting.ckey // Doing it on purpose will let the person control the blank body
+				to_chat(G.affecting, "<span class='biggerdanger'>ARGHHH--</span>")
+		else
+			to_chat(user, "<span class='warning'>Come here yourself!</span>")
+			return*/
```

Comment:
Another big block of commented out code added.

You need to have really, really good reason to leave in non-functional, commented out feature. Else it is codebloat that another person will have to figure out why did you add and when to uncomment them

Reply:
I have added these features to see what people would think of them, though I realize now I should have been more clear than just saying "added in some experimental commented out features".